fabric-registry-sync-v0/src/main/java/net/fabricmc/fabric/impl/registry/sync/entrylists/defaults/DefaultCustomRegistryEntryListsImpl.java

```diff
@@ -0,0 +1,78 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.impl.registry.sync.entrylists.defaults;
+
+import java.util.List;
+import java.util.Objects;
+
+import net.minecraft.registry.RegistryEntryLookup;
+import net.minecraft.registry.entry.RegistryEntryList;
+
+import net.fabricmc.fabric.impl.registry.sync.entrylists.CustomRegistryEntryListSerializerImpl;
+import net.fabricmc.fabric.impl.registry.sync.entrylists.util.RegistryWrapperUtils;
+
+public class DefaultCustomRegistryEntryListsImpl {
+	private static boolean initialized = false;
+
+	public static void register() {
+		if (initialized) {
+			return;
+		}
+
+		initialized = true;
+		CustomRegistryEntryListSerializerImpl.registerSerializer(UnionRegistryEntryList.SERIALIZER);
+		CustomRegistryEntryListSerializerImpl.registerSerializer(IntersectionRegistryEntryList.SERIALIZER);
+		CustomRegistryEntryListSerializerImpl.registerSerializer(InverseRegistryEntryList.SERIALIZER);
+		CustomRegistryEntryListSerializerImpl.registerSerializer(UniversalRegistryEntryList.SERIALIZER);
+	}
```

Comment:
It would be cleaner to move this to `FabricRegistryInit` directly and get rid of the unnecessary `initialized` field